framework/source/class/qx/ui/table/cellrenderer/AbstractImage.js

```diff
@@ -232,15 +232,15 @@ qx.Class.define("qx.ui.table.cellrenderer.AbstractImage",
     _getContentHtml : function(cellInfo)
     {
       var content = "<div></div>";
-
-      // background size is critical for high-resolution images
-      var backgroundSize = this.__imageData.width + "px " + this.__imageData.height + "px";
+      var backgroundSize = "auto auto";
```

Comment:
The original code did not use background-size.

See here: https://github.com/qooxdoo/qooxdoo/commit/0937f6c9ef8bf14ca05a9f778001b8d9290b0804#diff-8fdc282bf7f5c55d884a72d38bd81948

Reply:
Yes but background-size: auto auto; is the default value. The code would become more complicated building two different style objects (one with and one without background-size).

Reply:
OK